imod/mf6/multimodel/modelsplitter.py

```diff
@@ -94,7 +94,7 @@ def slice_model(partition_info: PartitionInfo, model: Modflow6Model) -> Modflow6
         # to drop these packages. Create schemata dict only containing the
         # variables with a AllNoDataSchema.
         allnodata_schemata = filter_schemata_dict(
-            package._write_schemata, (AllNoDataSchema)
+            package._write_schemata, (AllNoDataSchema, IndexesSchema)
```

Comment:
update the comment above, now it only mentions AllNoDataSchema. The variable name allnodata_schemata can also be renamed appropriately

Reply:
I added some comment, and refactored the crucial logic of IndexesSchema into the EmptyIndexesSchema, the latter object doesn't check for monotonicity, which is what is what is required here.